CHANGELOG.md

```diff
@@ -86,7 +86,8 @@ a parent Component are accessible this way. The Model's typed %Sets and `add####
 are no longer necessary to compose a Model, since any Component can now be composed of
 components. `Model` still supports `addd####()` methods and de/serialization of Sets,
 but components added via `addComponent` are NOT included in the Sets but contained
-in the Component's *components* property list. Details in PR#1014.
+in the Component's *components* property list. Details in PR#1014. **Note**, it is now
+strictly required that immediate subcomponents of the same type have unique names. For example, a Model's `BodySet` cannot contain two bodies named *tibia* because it is  ambiguous as to which *tibia* `Body` a `Joint` or any other component is referring to.
```

Comment:
Could this Note be moved high up under `Converting from v3.x to v4.0`? Also, isn't the requirement even stronger now? Subcomponents of must have unique names, regardless of type?

Reply:
Yes. Will update.